src/forms/BookingDatesForm/BookingDatesForm.test.js

```diff
@@ -13,6 +13,16 @@ import EstimatedBreakdownMaybe from './EstimatedBreakdownMaybe';
 const { Money } = sdkTypes;
 
 const noop = () => null;
+const lineItems = [
+  {
+    code: 'line-item/nights',
```

Comment:
Should this be changed? At least the other code uses singular form.

```suggestion
    code: 'line-item/night',
```